recipes/arpeggio/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "Arpeggio" %}
+{% set version = "1.9.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 948ce06163a48a72c97f4fe79ad3d1c1330b6fec4f22ece182fb60ef60bd022b
+
+build:
```

Comment:
Looks like this could be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python